lite-rpc/src/bridge.rs

```diff
@@ -65,10 +70,11 @@ lazy_static::lazy_static! {
 /// A bridge between clients and tpu
 pub struct LiteBridge {
     pub rpc_client: Arc<RpcClient>,
+    pub faithful_rpc_client: Option<Arc<RpcClient>>,
```

Comment:
faithfull service use Solana RPC client to get the data?

Reply:
Yes It should work with current RPCClient, any issues will open PR on faithful side. Recently I had created a fix for the same issue on faithful side.

Reply:
Ok, I didn't know. But they don't implement all RPCclient API call?